server/base/src/main/java/org/apache/accumulo/server/util/MasterMetadataUtil.java

```diff
@@ -253,4 +253,27 @@ public static void updateTabletDataFile(ServerContext context, KeyExtent extent,
 
     tablet.mutate();
   }
+
+  public static void updateLastLocation(ServerContext context, KeyExtent extent, FileRef path, DataFileValue dfv, MetadataTime time,
+                                        String address, ZooLock zooLock, TServerInstance lastLocation) {
+
+    TabletMutator tablet = context.getAmple().mutateTablet(extent);
+    if(dfv.getNumEntries() > 0) {
+      //these two lines might not be needed for updating the last location
+      tablet.putFile(path, dfv);
```

Comment:
What files are we changing?  We are only changing the last location for the tablet, not the files.  The time thing however may be needed.

Reply:
I figured it wasn't needed but I did not want to break anything.